src/build-change-graph.js

```diff
@@ -33,7 +33,9 @@ async function getPackageChangedFiles({
 
   let dirtyChanges = await git(['status', '--porcelain', '--untracked-files', packageCwd], options);
 
-  dirtyChanges = getLinesFromOutput(dirtyChanges).map(line => line.substr(3));
+  // if filename has space like `sample index.js`, if its modified and uncommited, that file will have double quotes in git status
+  // example: '"packages/package-a/sample index.js"'. We need to strip `"` for that reason.
+  dirtyChanges = getLinesFromOutput(dirtyChanges).map(line => line.substr(3).replace(/"/g, ''));
```

Comment:
Last request: Can you do

```
dirtyChanges = getLinesFromOutput(dirtyChanges).map(line => line.substr(3));

dirtyChanges = dirtyChanges.replace(/"/g, ''));
```

so that its clear which part of the code the comment is explaining.

Reply:
> Last request: Can you do
> 
> ```
> dirtyChanges = getLinesFromOutput(dirtyChanges).map(line => line.substr(3));
> 
> dirtyChanges = dirtyChanges.replace(/"/g, ''));
> ```
> 
> so that its clear which part of the code the comment is explaining.

I didn't want to do map again on array

Reply:
Oh I didn't see the map. We could have also multilined the map callback and done it there too.

Reply:
> Oh I didn't see the map. We could have also multilined the map callback and done it there too.

should I fix it?

Reply:
I didn't want to bother you :) https://github.com/CrowdStrike/monorepo-next/pull/472